src/pages/userstats.js

```diff
@@ -397,10 +373,11 @@ export default function UserStats() {
   }
 
   // background opacity ranges from 0.0 to 1.0, loudness ranges from about -65 to 0 decibels, so 1.0 opacity == loud
-  function computeBackgroundOpacity() {
+  function computeCheeksOpacity() {
     let loudnessArray = audioFeatures.map((item) => item.loudness);
     let averageLoudness = average(loudnessArray);
-    return (averageLoudness + 65) * (1 / 65);
+    console.log(averageLoudness);
```

Comment:
left this here incase u wanna compare loudness values to decide if we still wanna do loudness